### DISCUSSION TITLE
nppe.glitch.me Legit? Not on this repo..

### DISCUSSION THREAD

**Discussion by yodaluca23** [3kh0/ext-remover#539] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I wanna try out using https://nppe.glitch.me/ (Not on this repo) but the data that you upload to it, can contain cookies and logins, and stuff... is the file uploaded to their servers? or is it all local? if it does upload, then has anyone used this? Is it safe?

**Comment by rwg100** [comment, marked as answer]
its completely safe. it doesnt even matter, since its the wifi data. none of the data from your chromebook to going through that site besides the netlog file you upload. and yes, its local.

**Reply by obkduishinijf** [reply]
@yodaluca23 mark as answer
